content/go/go.md

```diff
@@ -9,4 +9,4 @@ CatalogContent:
 
 Go (also referred to as "Golang") is an open-source compiled programming language that was designed at Google by Robert Griesemer, Rob Pike, and Ken Thompson. It had its initial release for the public in 2012.
 
-Go features known concepts such as static typing, memory safety, garbage collection, and more. It was originally designed to write high performance networking and infrastructure-relevant systems. Nowadays, it's used for cloud-based or server-side apps, DevOps, AI, Data Science, and more. Golang is known for its readability, simplicity, shallow learning curve, and great performance.
+Go features known concepts such as static typing, memory safety, garbage collection, and more. It was originally designed to write high performance networking and infrastructure-relevant systems. Nowadays, it's used for [cloud-based](https://www.codecademy.com/resources/docs/general/cloud-computing) or server-side apps, DevOps, AI, Data Science, and more. Golang is known for its readability, simplicity, shallow learning curve, and great performance.
```

Comment:
Wondering if we should file new entry issues for Go concepts like static typing, memory, and garbage collection?

Reply:
@Dusch4593 sounds like a good idea.